src/js/containers/ActiveZoneSelector/ActiveZoneSelector.js

```diff
@@ -40,18 +40,25 @@ class ActiveZoneSelector extends Component {
     }
 
     render() {
-        let { activeZone, intl, zoneList } = this.props;
+        let { activeZone, zoneList } = this.props;
         let zones = _.values(zoneList).map(zone => {
             return { value: zone.name, label: zone.name };
         });
 
+        let singleZoneStyle = { margin: "9px auto 0" };
+        let listZoneStyle = { margin: "3px auto 0" };
+
+        let activeZoneName = activeZone.name
```

Comment:
Still need to update `mapStateToProps` to just pull the `activeZone.name` since thats all we use in this component.